packages/playwright-ct-core/src/tsxTransform.ts

```diff
@@ -141,6 +143,23 @@ function collectJsxComponentUsages(node: T.Node): Set<string> {
   return names;
 }
 
+function collectClassMountUsages(node: T.Node): Set<string> {
+  const names = new Set<string>();
+  traverse(node, {
+    enter: p => {
+      // Treat calls to mount and all identifiers in arguments as component usages e.g. mount(Component)
+      if (t.isCallExpression(p.node) && t.isIdentifier(p.node.callee) && p.node.callee.name === 'mount') {
```

Comment:
We used to have this as you probably know. But the `isIdentifier(p.node.callee)` felt overly restrictive and limited us a couple of times.

Reply:
I don't see how? What approach would you recommend to address this? In my opinion, relying solely on file extension is not a viable option.